core-services/prow/02_config/openshift/coredns-ocp-dnsnameresolver/_prowconfig.yaml

```diff
@@ -0,0 +1,14 @@
+tide:
+  queries:
+  - labels:
```

Comment:
What's the reason these other labels are not included?
```
    labels:
    - approved
    - backport-risk-assessed
    - cherry-pick-approved
    - jira/valid-bug
    - jira/valid-reference                                                                     
    - lgtm
```

Reply:
Thanks for pointing that out. I have now used the prow config for tide from cluster-dns-operator directory. The reference for each field can be found here: https://docs.prow.k8s.io/docs/components/core/tide/config/#queries

Reply:
I noticed the simple labeling strategy too but I didn't ask to align it with the other NetEdge repos for the reason of simplifying the initial development process. However here I may be talking about 3 labels more: docs/qe/px approved.

Reply:
I see. I don't mind if you want add the label requirements now, or later. But I think we can apply all the labels we need manually if you want to shortcut the labels during development.

Reply:
I have added those labels here: https://github.com/openshift/release/pull/46743/files#diff-ba74bbf69def205584786687a569bfccd6b731599b4987bc49f2c9f68af107c4R71-R77